src/accessibility.md.njk

```diff
@@ -56,9 +56,9 @@ The Design System website at [https://design-system.service.gov.uk/](https://des
 
 We plan to fix this accessibility issue by the end of September 2021.
 
-The GOV.UK Frontend documentation website at http://frontend.design-system.service.gov.uk/ is partially compliant because of [issues caused by our Technical Documentation Template](https://tdt-documentation.london.cloudapps.digital/accessibility/#using-the-technical-documentation-template-for-your-own-documentation).
+The GOV.UK Frontend documentation website at http://frontend.design-system.service.gov.uk/ is partially compliant because of [an issue caused by our Technical Documentation Template](https://tdt-documentation.london.cloudapps.digital/accessibility/#using-the-technical-documentation-template-for-your-own-documentation).
 
-We plan to fix these accessibility issues by the end of July 2021.
+We are currently working on a plan to fix this accessibility issue.
```

Comment:
This text will serve as a placeholder we can iterate on.

Are we required to say _when_ we intend to fix the issue by?

Reply:
```suggestion
We plan to fix this issue by 30 September 2021
```
Basing this ^ on line 70 here: https://github.com/alphagov/tdt-documentation/pull/163/files

Reply:
Ok, we're not to include a date for fixes.